front-end/src/Routers/index.jsx

```diff
@@ -4,14 +4,16 @@ import {
   Switch,
   Route
 } from 'react-router-dom';
-import { Login } from '../Pages/index'; 
+import { Login } from '../Pages/index';
+import { ClientProduct } from '../Pages/index'
```

Comment:
Acredito que de para usar no mesmo destructuring
```suggestion
import { Login, ClientProduct } from '../Pages/index';
```